app/views/admin/lessons.html.erb

```diff
@@ -1,2 +1 @@
-<h1>Admin#lessons</h1>
-<p>Find me in app/views/admin/lessons.html.erb</p>
+<%= react_component('LessonsPage', {lessons: @lessons}) %>
```

Comment:
sorry, super nitpicky but can you add spaces after/before bracket so { lessons: @lessons }